### PR TITLE
[BACKLOG-31898] CSRF - /ServiceAction

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/millenniumfalcon please review

pentaho-ajax and pentaho-csrf implementation was centralized in common-ui repo. The other projects now use this implementation instead of their own.

 - https://github.com/pentaho/data-access/pull/1081
 - https://github.com/pentaho/pentaho-platform-plugin-dashboards/pull/512
 - https://github.com/pentaho/pentaho-platform/pull/4588
 - https://github.com/pentaho/pentaho-platform-ee/pull/1368

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 31s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **5259**, Failures: **0**, Skipped: **3** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/common-ui/job/PR-1487/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: impl/client/src/main/javascript/web/repo/pentaho-ajax.js [hunk 1]
```diff
@@ -14,334 +14,309 @@
 * limitations under the License.
 *
 */
-define("common-repo/pentaho-ajax",[], function(){});
+define("common-repo/pentaho-ajax",["common-repo/pentaho-csrf"], function(){});
+
+// Project: pentaho-platform-plugin-common-ui
+
+/* globals CONTEXT_PATH, FULL_QUALIFIED_URL, WEB_CONTEXT_BASE */
+
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/repo/pentaho-ajax.js]: Almost exactly what I meant in the dashboards review :-)
Don't need to create a sync csrf/service, though.

**Comment by buildguy** [comment]
### :warning: Build finished in 4m 21s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 2 test(s) failed:

<details><summary><b>unit.PPP-3455.PPP-3455 mql.submitXmlQuery() should parse valid response</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>unit.PPP-3455.PPP-3455 mql.submit() should parse valid response</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

  
Tests run: **5259**, Failures: **2**, Skipped: **3** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/common-ui/job/PR-1487/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 16s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **5259**, Failures: **0**, Skipped: **3** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/common-ui/job/PR-1487/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by davidmsantos90** [comment]
@pentaho/millenniumfalcon pr updated. please review

**Pull request merged**